### PR TITLE
stack exchange/overflow buttons

### PR DESCRIPTION
**Pull request by johnlane**
I created a buttons for Stack Exchange and Stack Overflow.

### CODE REVIEW

**Comment by lipis** [comment]
Not sure if I'm going to add more at the moment.. but from where did you get the colors..? can you provide the source..?

**Comment by johnlane** [comment]
I just took the colours from the logos on the web sites. I took a screen shot and used the dropper in Gimp to select mid-range colours from the logos. Nothing particularly scientific.

**Comment by meeDamian** [comment]
@lipis can you elaborate on:

> Not sure if I'm going to add more at the moment..

I'd like to PR with a `Behance` button, but I'm not sure if that makes sense any more, does it?

**Comment by lipis** [comment]
@chester1000 My main reason is because it's super easy to add any new button you want if you are using Less or SCSS.. If you are not using any of those.. you probably should.. but that's another story.

The second reason is where should I stop?! How many buttons should I possibly add into that project?!

I could maybe add as many as possible (until we'll run out of FA icons, but maybe then add other icon vendors) but then it's going to be too much deleting for people that only want a few.. :)

On the other hand.. with this project I also wanted to make sure that all the colors that are used.. are as official as possible :) So maybe that's a good reason to just add more buttons to have it at least as a reference for the colors.. but I just realised that there is this website: http://brandcolors.net/

So I'm not sure :) What do you think? Should I just continue adding more and more buttons?! Is it really needed?

**Comment by meeDamian** [comment]
**tl;dr: IMO the more buttons the better.**

@lipis Your rules are a great reason to add as many buttons as possible ;). I agree it's easy to **_write_** new button definitions, but first you need to check if icon exists in `fa` or `glyphicon`, decide on a proper bkg and font color, etc...  agreed it's not **super** hard to do that, but still it will be great to have one repository of standardized, scalable, good quality and consistent social buttons.

I wouldn't worry too much about adding unnecessary bytes to styles, because:
- the more buttons there is the more of them will overlap, so instead of two definitions there will be one definition with two class names,
- even with hundreds of buttons size of this library will be negligible,
- there's plenty of tools that strip unnecessary styles for production.

**Comment by lipis** [comment]
> even with hundreds of buttons size of this library will be negligible

Not exactly, as every new button adds a bit less than 1KB to the minified CSS, it won't be exactly negligible..

But maybe after all I'll start adding more just to have them as references.. haven't make up my mind yet.. and it's a bit late here :) Will think about it.. maybe I'll add one file that has only the top 3 vendors and all of them in the rest.. :)

**Comment by mathieumg** [comment]
:+1: +1
